### PR TITLE
Add 'kw-completion'

### PR DESCRIPTION
**Pull request by shayenne**
Add autocomplete for kw commands whole words.
Update setup.sh to add source for the new file.

Signed-off-by: Shayenne da Luz Moura <shayenneluzmoura@gmail.com>

### CODE REVIEW

File: kw-completion.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+complete -W "export build install prepare new ssh mail mount
+	     umount boot vars up codestyle maintainers help" kw
```
Comment (rodrigosiqueira) [inline kw-completion.sh]: I recommend you to put this line in the file "kw.sh" on top of "function kw()", with this change you can get rid of the file "kw-completion.sh". Additionally, I recommend you add the short commands in the list as well.

**Comment by rodrigosiqueira** [comment]
First of all, thanks for your patch! This is a really nice pacth :)

Follows my comments in topics:

* You don't need the kw-completion.sh, take a look on my inline comment.
* Could you check if the "comple" command is a default? If not, could you update the readme by add this dependency?
* Update your commit message because it is not clear. The message "Add 'kw-completion", probably could be replaced by something like "Add autocomplete for kw". Finally, describe a little the changes in the commit message.

File: kw-completion.sh (hunk 1 above)
Comment (shayenne) [reply kw-completion.sh]: Okay! Regarding the complete command, I found that it is a builtin function. Does it need to add in the dependence?

Comment (shayenne) [reply kw-completion.sh]: https://www.gnu.org/software/bash/manual/html_node/Programmable-Completion-Builtins.html

**Comment by rodrigosiqueira** [comment]
Merged:

https://github.com/rodrigosiqueira/kworkflow/commit/bc43e82dfa46f4b7830d017a4bda7bffc3f5610d

**Pull request closed without merging**